config/routes.rb

```diff
@@ -18,6 +18,8 @@
 
   get '/profile', to: 'users#show'
 
+  get '/search', to: 'search#items'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.